core/common/src/main/java/alluxio/Constants.java

```diff
@@ -206,5 +206,10 @@
   public static final String TABLE_MASTER_CLIENT_SERVICE_NAME = "TableMasterClient";
   public static final long TABLE_MASTER_CLIENT_SERVICE_VERSION = 1;
 
+  // Medium name
+  public static final String MEDIUM_MEM = "MEM";
+  public static final String MEDIUM_SSD = "SSD";
+  public static final String MEDIUM_HDD = "HDD";
```

Comment:
@LuQQiu @ggezer  shall we create a static class `Medium.java` and make three static member String constants called `MEM`, `SSD`, `HDD` ?  I slightly incline for this approach as we can make more fancy changes in the future in this class. But I am also cool for the current solution.. I will leave this up to you